api/src/main/java/liquibase/ext/sqlgenerator/core/ModifyColumnGenerator.java

```diff
@@ -192,12 +192,10 @@ String getPostDataTypeString(Database database) {
 	String getDefaultClause(ColumnConfig column, Database database) {
 		String clause = "";
 		String defaultValue = column.getDefaultValue();
-		if (defaultValue != null) {
-			if (database instanceof MySQLDatabase) {
+		if (defaultValue != null && database instanceof MySQLDatabase){
 				clause += " DEFAULT " + DataTypeFactory.getInstance().fromObject(defaultValue, database)
 					.objectToSql(defaultValue, database);
 			}
```

Comment:
Here, perhaps too much by one tab.

Reply:
Yes its should be like that, do you want to introduce a new braces

Reply:
No extra curly braces. The point is that there are two tabs before "if" and three tabs before the brace associated with this "if". Or GitHub shows something different than what you see in your IDE.